src/ethereum_spec_tools/forks.py

```diff
@@ -40,13 +40,14 @@ def discover(cls: Type[H]) -> List[H]:
         for pkg in modules:
             mod = importlib.import_module(pkg.name)
             block = getattr(mod, "MAINNET_FORK_BLOCK", -1)
+            timestamp = getattr(mod, "MAINNET_FORK_TIMESTAMP", -1)
```

Comment:
This might be a better way to differentiate between not having a fork start and it being `None`:

```suggestion
            try:
                block = getattr(mod, "MAINNET_FORK_BLOCK")
                timestamp = getattr(mod, "MAINNET_FORK_TIMESTAMP")
            except AttributeError:
                continue
```